### PR TITLE
Add new optional field SeverityLevel to Service Alerts feed

### PR DESCRIPTION
**Pull request by ibi-group-team**
This is a proposal to add a new field SeverityLevel to the ServiceAlerts feed. The SeverityLevel field would be optional and comprised of enum values:
- UNKNOWN_SEVERITY
- INFO
- WARNING
- SEVERE

See discussion here: https://github.com/google/transit/issues/133

Announced on the GTFS-realtime Google Groups here: https://groups.google.com/forum/#!topic/gtfs-realtime/t8bpU5gbjDU

### CODE REVIEW

**Comment by barbeau** [comment]
I also have a preference for `INFO` but no strong objection to `INFORMATIONAL`

**Comment by abyrd** [comment]
Likewise I also have a preference for `INFO` over `INFORMATIONAL`. I also wouldn't actively object to `INFORMATIONAL` but there seem to be more people who support `INFO` / `WARNING` / `SEVERE`.

**Comment by abyrd** [comment]
Initially I had a preference for `INFO` over `INFORMATIONAL`. But on further reflection I think this is just because the value `INFO` is often used in logging, which is really a different application. I can see why we'd want to avoid abbreviations in enum values. `INFO` in this case may stand for `INFORMATIONAL`, or does it stand for `INFORMATION`? Either way the grammatical parallelism with the other enum values is questionable.

I think the main argument for `INFO` is that it is concise and not strictly an adjective or a noun. It seems like more people prefer `INFO` but I wouldn't actively oppose `INFORMATIONAL`.

**Comment by ibi-group-team** [comment]
This pull request has been open for more than one week, so we are calling for a vote.

Voting will be closed on Thursday January 31 at 23:59:59 UTC.

**Comment by laurentg** [comment]
Just as a side note, in our Urbiplan alerts product we already have this severity level. We use 3 levels: INFO, WARN, CRITICAL. +1

**Comment by skinkie** [comment]
+1

**Comment by paulswartz** [comment]
+1

**Comment by gcamp** [comment]
+1

File: gtfs-realtime/proto/gtfs-realtime.proto [hunk 1]
```diff
@@ -427,6 +427,17 @@ message Alert {
   // Full description for the alert as plain-text. The information in the
   // description should add to the information of the header.
   optional TranslatedString description_text = 11;
+  
+  // Severity of this alert. This field is still experimental, 
+  // and subject to change. It may be formally adopted in the future.
```
Comment (barbeau) [inline gtfs-realtime/proto/gtfs-realtime.proto]: This field needs to be marked as experimental in the .proto file too.

File: gtfs-realtime/spec/en/reference.md [hunk 2]
```diff
@@ -66,6 +66,7 @@ Fields labeled as **experimental** are subject to change and not yet formally ad
             *   [Effect](#enum-effect)
             *   [TranslatedString](#message-translatedstring)
                 *   [Translation](#message-translation)
+			*	[SeverityLevel](#enum-severitylevel)
```
Comment (barbeau) [inline gtfs-realtime/spec/en/reference.md]: Could you please match the whitespace indenting here to the same level as `TranslatedString`?  It renders right on GitHub but the Markdown should still be formatted consistently.

**Comment by barbeau** [comment]
+1

@ibi-group-team There are a few formatting items I commented on that need to be fixed prior to merge.

Also, as a side note, the conventions proposed here match Google's severity levels (Critical, Warning, Informational) inferred from Alert Effects as listed on https://support.google.com/transitpartners/answer/6374472?hl=en&ref_topic=6159819.

File: gtfs-realtime/proto/gtfs-realtime.proto (hunk 1 above)
Comment (barbeau) [reply gtfs-realtime/proto/gtfs-realtime.proto]: Whoops, I see that you did include this.  Could you move the "experimental" comment to it's own line to match the others?

**Comment by prhod** [comment]
+1

File: gtfs-realtime/spec/en/reference.md (hunk 2 above)
Comment (ibi-group-team) [reply gtfs-realtime/spec/en/reference.md]: Can you please clarify what you mean by this? If you view the full markdown file, the indentation is formatted correctly.

**Comment by ibi-group-team** [comment]
Voting for this proposal is closed and the results are:
Yes: 8
No: 0

So the proposal passes! We'll get this merged.

Thank you all!

**Comment by barbeau** [comment]
Thanks @ibi-group-team and all!

**Pull request merged**